addons/packages/grafana/metadata.yaml

```diff
@@ -9,8 +9,8 @@ spec:
   shortDescription: "Visualization and analytics software"
   providerName: VMware
   maintainers:
-  - name: Guanpeng Gao
+  - name: XiaoShuang Guo
   categories:
     - "monitoring"
     - "observability"
-  supportDescription: "The detailed documentation is on https://grafana.com/.  Support is available from VMware. Refer to the VMware Product Guide for specific support terms. "
+  supportDescription: "The detailed documentation is on https://grafana.com/.  Support is available from VMware. Refer to the VMware Product Guide for specific support terms. "
```

Comment:
Can you add a newline at the end of this file? Otherwise the Yaml Lint check will fail.